paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -284,6 +289,7 @@ internal data class PaymentMethodMetadata(
                 financialConnectionsAvailable = isFinancialConnectionsAvailable(),
                 paymentMethodSaveConsentBehavior = paymentMethodSaveConsentBehavior,
                 linkMode = elementsSession.linkSettings?.linkMode,
+                paymentMethodIncentive = null,
```

Comment:
Why is this null? We have elementsSession, I'd think we could pipe the data through.

Reply:
This is for CustomerSheet, which doesn’t offer Instant Debits as a payment method.